backend/payments/serializers.py

```
@@ -1,7 +1,166 @@
+        return super().create(validated_data)
+
+    def update(self, instance, validated_data):
+        cashier_id = validated_data.pop('cashier_id', None)
+        request = self.context.get('request')
+        if cashier_id is not None:
+            role = get_role(request.user) if request and getattr(request, 'user', None) else None
+            if role != 'Admin':
+                from rest_framework.exceptions import PermissionDenied
+                raise PermissionDenied('Only admin can reassign cashier')
+
+    def create(self, validated_data):
```

Ruff (0.14.8)

Redefinition of unused `create` from line 68

(F811)